### PR TITLE
Update registrar default queue name

### PR DESCRIPTION
**Pull request by iloveagent57**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Review by kdmccormick — COMMENTED** [review]
https://github.com/edx/registrar/pull/122#issuecomment-499136499

**Review by kdmccormick — APPROVED** [review]
May want to beef up the changelog entry.

Otherwise :guitar:

File: CHANGELOG.md [hunk 1]
```diff
@@ -768,3 +768,6 @@
 
 - Role: supervisor
   - Add registrar to `pre_supervisor_checks.py`
+
+- Role: registrar
```
Comment (jdmulloy) [inline CHANGELOG.md]: Newest changelog entries should be on the top.

**Review by jdmulloy — CHANGES_REQUESTED** [review]

**Review by jdmulloy — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (kdmccormick) [reply CHANGELOG.md]: @jdmulloy wait, really? I'd add a note to the Changelog indicating that's the case. I've been adding changes to the end of the file for months now.

**Pull request merged**